packages/connector/src/types.ts

```diff
@@ -87,7 +87,7 @@ export type ProofQuery = {
   claimId?: string;
   credentialSubjectId?: string;
   context?: string;
-  type?: string;
+  type?: string[];
```

Comment:
Can iden3 really handle multiple types right now?

Reply:
it's the way issuer build type property in VC